concent_api/core/admin.py

```diff
@@ -16,9 +27,53 @@
 from .models import StoredMessage
 from .models import Subtask
 from .models import SubtaskWithTimingColumnsManager
+from .payments.report import create_report
 
 ACTIVE_STATE_NAMES = [x.name for x in Subtask.ACTIVE_STATES]
 PASSIVE_STATE_NAMES = [x.name for x in Subtask.PASSIVE_STATES]
+DECORATORS = [transaction.non_atomic_requests(using='control')]
+
+
+@method_decorator(DECORATORS, name='dispatch')
+class PaymentReportAdminView(TemplateView):
+
+    template_name = 'admin/payment_report.html'
+
+    def get_context_data(self, **kwargs: Any) -> Dict:
+        context_data = super().get_context_data(**kwargs)
+        context_data['payment_report'] = create_report()
+        context_data['eth_network'] = self._get_eth_network()
+        return context_data
+
+    @staticmethod
+    def _get_eth_network() -> str:
+        session = requests.Session()
+        data = json.dumps({'id': 67, 'jsonrpc': '2.0', 'method': 'net_version', 'params': []})
+        headers = {'Content-type': 'application/json'}
```

Comment:
Those two lines above are good candidates for a constants.

Reply:
Done